deploy/crds/operator.ibm.com_v1alpha1_metaoperatorcatalog_cr.yaml

```diff
@@ -20,3 +20,11 @@ spec:
     sourceNamespace: openshift-marketplace
     targetNamespaces:
       - ibm-licensing-operator
+  - name: ibm-mongodb-operator
+    namespace: ibm-mongodb-operator
+    channel: alpha
+    packageName: ibm-mongodb-operator-app
+    sourceName: opencloud-operators
+    sourceNamespace: openshift-marketplace
+    targetNamespaces:
```

Comment:
I think we need to update the example here by adding `description` field and remove `targetNamespaces` for all operators in `MetaOperatorCatalog`

Reply:
👌， I'll update all the onboarded operator